e2e/test.js

```diff
@@ -0,0 +1,51 @@
+const expect = require('chai').expect;
+
+describe('InstantSearch e-commerce demo', () => {
+  before(() => {
+    browser.maximizeWindow();
+  });
+
+  it('must work', async () => {
+    await browser.url(
+      'https://deploy-preview-3967--instantsearchjs.netlify.com/examples/e-commerce/'
+    );
+
+    // Select "Apple" brand in list
+    const brand = await browser.$('span=Apple');
```

Comment:
is there a reason this isn't `querySelector`?

Reply:
We are not in the browser context here so we have to use the selector from WebdriverIO. It is a bit different than `querySelector` https://webdriver.io/docs/selectors.html